Documentation/operations/metrics.rst

```diff
@@ -190,93 +190,92 @@ Exported Metrics
-
+============================================= ================================================== ========== ========================================================
+Name                                          Labels                                             Default    Description
+============================================= ================================================== ========== ========================================================
+``datapath_conntrack_dump_resets_total``      ``area``, ``name``, ``family``                     Enabled    Number of conntrack dump resets. Happens when a BPF entry gets removed while dumping the map is in progress.
+``datapath_conntrack_gc_runs_total``          ``status``                                         Enabled    Number of times that the conntrack garbage collector process was run
+``datapath_conntrack_gc_key_fallbacks_total``                                                    Enabled    The number of alive and deleted conntrack entries at the end of a garbage collector run labeled by datapath family
+``datapath_conntrack_gc_entries``             ``family``                                         Enabled    The number of alive and deleted conntrack entries at the end of a garbage collector run
+``datapath_conntrack_gc_duration_seconds``    ``status``                                         Enabled    Duration in seconds of the garbage collector process
+============================================= ================================================== ========== ========================================================
 IPSec
 ~~~~~
 
-============================================= ================================================== ========================================================
-Name                                          Labels                                             Description
-============================================= ================================================== ========================================================
-``ipsec_xfrm_error``                          ``error``, ``type``                                Total number of xfrm errors.
-============================================= ================================================== ========================================================
+============================================= ================================================== ========== ========================================================
+Name                                          Labels                                             Default    Description
+============================================= ================================================== ========== ========================================================
+``ipsec_xfrm_error``                          ``error``, ``type``                                Disabled   Total number of xfrm errors.
```

Comment:
I wonder if this is actually enabled by default but only if you configure ipsec :thinking:

Reply:
I used the default for installation. Didn't find the metric. Maybe I can try by configuring ipsec. Though this should mean this is correctly marked as `Disabled` 🤔

Reply:
grepping around for the xfrm metrics, I found the XFRM Collector here:

```
$ git grep NewXFRMCollector
pkg/datapath/linux/ipsec/xfrm_collector.go:// NewXFRMCollector returns a new prometheus.Collector for /proc/net/xfrm_stat
pkg/datapath/linux/ipsec/xfrm_collector.go:func NewXFRMCollector() prometheus.Collector {
pkg/datapath/linux/node.go:             ipsecMetricCollector:   ipsec.NewXFRMCollector(),
```

And then tracing up a bit further, we see it being conditionally enabled here:

https://github.com/cilium/cilium/blob/15cc267f70c7e00fee784a992c806dbc3681fcc5/pkg/datapath/linux/node.go#L1560-L1578

So it's enabled by default, but only if you're using the relevant feature. In a way, this is similar to the way that the vector metrics work. They're enabled by default, but they won't always show up. They'll only show up when you start to use the feature that emits the metric.

Reply:
Thanks for this find @joestringer